### PR TITLE
[WICKED] Startstop test case 3

### PR DESCRIPTION
**Pull request by jlausuch**
Test 3  : Bridge - ifup, remove all config, ifreload

- Related ticket: https://progress.opensuse.org/issues/41657
- Verification run: http://fromm.arch.suse.de/tests/1972#

### CODE REVIEW

File: tests/wicked/startandstop_sut.pm [hunk 1]
```diff
@@ -72,6 +72,17 @@ sub run {
     $self->cleanup($config, "br0");
     $self->cleanup($dummy,  "dummy0");
 
+    $self->before_scenario('Test 3', 'Bridge - ifup, remove all config, ifreload', $iface);
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run("rm /etc/sysconfig/network/ifcfg-$iface $config $dummy");
+    assert_script_run("wicked ifreload --timeout infinite all");
+    my $res = script_output("ip link|grep 'dummy0\|br0'||true");
```
Comment (asmorodskyi) [inline tests/wicked/startandstop_sut.pm]: this PR already hit osd https://github.com/os-autoinst/os-autoinst/pull/1030 so you may also use this 
`my $res = script_output("ip link|grep 'dummy0\|br0'",90, proceed_on_failure => 1);`

Comment (cfconrad) [reply tests/wicked/startandstop_sut.pm]: you could also use just script_run()

Comment (asmorodskyi) [reply tests/wicked/startandstop_sut.pm]: yeah actually in this case probably better just to use `script_run`

Comment (jlausuch) [reply tests/wicked/startandstop_sut.pm]: done

File: tests/wicked/startandstop_sut.pm [hunk 2]
```diff
@@ -72,6 +72,17 @@ sub run {
...
     $self->cleanup($dummy,  "dummy0");
 
+    $self->before_scenario('Test 3', 'Bridge - ifup, remove all config, ifreload', $iface);
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run("rm /etc/sysconfig/network/ifcfg-$iface $config $dummy");
+    assert_script_run("wicked ifreload --timeout infinite all");
+    my $res = script_run("ip link|grep 'dummy0\|br0'");
+    $results{3} = $res ? "PASSED" : "FAILED";
```
Comment (asmorodskyi) [inline tests/wicked/startandstop_sut.pm]: grep will return 0 on success , right ?

**Pull request merged**